CRM/Admin/Form/Options.php

```diff
@@ -133,6 +133,13 @@ public function preProcess() {
     }
   }
 
+  /**
+   * Get the form-specific delete message.
+   */
+  public function setDeleteMessage(): void {
+    $this->deleteMessage = ts('WARNING: Deleting this option will result in the loss of all %1 related records which use the option.', [1 => $this->_gLabel]) . ts('This may mean the loss of a substantial amount of data, and the action cannot be undone.') . ts('Do you want to continue?');
```

Comment:
The earlier PR also removed some needed spacing so these sentences now run together. Also should this be against 5.77?
```suggestion
    $this->deleteMessage = ts('WARNING: Deleting this option will result in the loss of all %1 related records which use the option.', [1 => $this->_gLabel]) . ' ' . ts('This may mean the loss of a substantial amount of data, and the action cannot be undone.') . ' ' . ts('Do you want to continue?');
```